packages/transport/src/modules/devices/backbone/BackbonePostDevices.ts

```diff
@@ -3,11 +3,13 @@ import { IChallengeSignedSerialized } from "../../challenges/data/ChallengeSigne
 export interface BackbonePostDevicesRequest {
     devicePassword: string;
     signedChallenge: IChallengeSignedSerialized;
+    isBackupDevice?: boolean;
```

Comment:
At first glance I was confused why here `isBackupDevice` may be undefined. So I took a deeper look and saw that `sendDevice` is called by `CreateIdentityRecoveryKit` and `CreateDevice`. The former sets `isBackupDevice` to true while the latter doesn't set it. We could set it to false in the use case or have `sendDevice` set it to false if it is undefined, instead of letting the Backbone do that. Is there a coding guideline that suggests if such things should be done at a higher or lower level? If we do this at a higher level, it might make our code less complicated in some places.

Reply:
Is this so important? The backbone has a default, we take that default (false) when not sending anything.

Reply:
I don't consider it indispensable for the implementation here.

> Is there a coding guideline that suggests if such things should be done at a higher or lower level?

 This was meant as a general question. If there is an unambiguous suggestion, your question about the importance should probably be answered by the significance of code style we have in our repos in general.

Reply:
IMO `Backbone{Post|GET|...}...Request`s should be **as close** to what the backbone expects as possible.

Best example are the responses - we work with undefined for not set fields, the backbone sets null. If we only look on our codestyle everything blows up.

Theses Requests and Responses are defined by the backbone, so we do not really decide - also I don't see a really good reason to have this required.

Reply:
I'm not completely satisfied by "outcome" of this discussion, but I get the feeling that you do not wish to discuss this further. And as already stated, since I don't consider it indispensable to change something here, I'm okay with putting it to rest for now.